nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const router = express.Router();
+
+const meals = require('./../data/meals.json');
+
+router.get('/', async (request, response) => {
+    try {
+        let result;
+        if ('title' in request.query) {
+            const title = request.query.title;
+            const getTitle = meals.find((meal) =>
+                meal.title.toLowerCase().includes(title.toLowerCase()),
+            );
+            result = getTitle;
+        }
+
+        if ('maxPrice' in request.query) {
+            const maxPrice = Number(request.query.maxPrice);
+            const lessThanMaxPrice = meals.filter(
+                (meal) => meal.price < maxPrice,
+            );
+            result = lessThanMaxPrice;
+        }
+
+        if ('createdAfter' in request.query) {
+            const createdAfter = new Date(request.query.createdAfter);
+            const d1 = createdAfter.getTime();
+
+            const createdAfterDate = meals.filter((meal) => {
+                const availableDate = new Date(meal.createdAt);
+                const d2 = availableDate.getTime();
+                return d1 < d2;
+            });
+            result = createdAfterDate;
```

Comment:
why do you filter by time instead of by date? Can you make the correction and filter by date?